src/stdlib/parse_grok.rs

```diff
@@ -17,7 +17,12 @@ static MEZMO_PATTERNS: &[(&str, &str)] = &[
     ("OPTIONAL_SENTENCE", r#"[\p{L},":;\s\-]*"#),
     ("PSQL_LOG", "%{TIMESTAMP_ISO8601:timestamp} %{DATA} %{SQUARE_BRACKET}%{INT:pid}%{SQUARE_BRACKET} %{EMAILLOCALPART:username}%{NOTSPACE}%{WORD:database} %{GREEDYDATA:message}"),
     ("GOLANG_LOG", "%{DATESTAMP:timestamp} %{LOGLEVEL:level} %{GREEDYDATA:message}"),
-    ("ELASTICSEARCH_LOG", "%{SQUARE_BRACKET}%{TIMESTAMP_ISO8601:timestamp}%{SQUARE_BRACKET}%{SQUARE_BRACKET}%{LOGLEVEL:level}%{SPACE}%{SQUARE_BRACKET}%{SQUARE_BRACKET}%{DATA:source}%{SQUARE_BRACKET} %{GREEDYDATA:message}")
+    ("ELASTICSEARCH_LOG", "%{SQUARE_BRACKET}%{TIMESTAMP_ISO8601:timestamp}%{SQUARE_BRACKET}%{SQUARE_BRACKET}%{LOGLEVEL:level}%{SPACE}%{SQUARE_BRACKET}%{SQUARE_BRACKET}%{DATA:source}%{SQUARE_BRACKET} %{GREEDYDATA:message}"),
+    ("K8S_POD_NAME", r"[a-z0-9A-Z\-.]+"),
```

Comment:
Given that pod, namespace and container patterns are identical, can we define a single pattern and re-use to extract the k8s resource name?

Reply:
My thought process was each type is specific regardless of them matching via the same regex. That said, perhaps I could define something like a `%{NUMBERS_LETTERS_DASH}`to share amongst the three? I'm not sure how useful it'll be outside this context however.

Reply:
Each pattern has to be compiled at last once. 
We can definitely keep the existing names `K8S_POD_NAME`, `K8S_NAMESPACE`, `K8S_CONTAINER` but they should reference the same pattern